packages/graphql/tests/tck/connections/sorting.test.ts

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { gql } from "apollo-server";
+import type { DocumentNode } from "graphql";
+import { Neo4jGraphQL } from "../../../src";
+import { createJwtRequest } from "../../utils/create-jwt-request";
+import { formatCypher, translateQuery, formatParams } from "../utils/tck-test-utils";
+
+describe("Relationship Properties Cypher", () => {
+    let typeDefs: DocumentNode;
+    let neoSchema: Neo4jGraphQL;
+
+    beforeAll(() => {
+        typeDefs = gql`
+            type Movie {
+                title: String!
+                actors: [Actor!]! @relationship(type: "ACTED_IN", properties: "ActedIn", direction: IN)
+            }
+
+            type Actor {
+                name: String!
+                movies: [Movie!]! @relationship(type: "ACTED_IN", properties: "ActedIn", direction: OUT)
+            }
+
+            interface ActedIn {
+                screenTime: Int!
+                year: Int!
+            }
+        `;
+
+        neoSchema = new Neo4jGraphQL({
+            typeDefs,
+            config: { enableRegex: true },
+        });
+    });
+
+    test("Projecting node and relationship properties with no arguments", async () => {
+        const query = gql`
+            query {
+                moviesConnection(first: 5, sort: { title: ASC }) {
```

Comment:
Can we add a test also with `after` to ensure that this doesn't add in `SKIP` twice?

Reply:
That's a good point